app/scripts/modules/core/src/pipeline/config/PipelineRegistry.ts

```diff
@@ -109,10 +118,6 @@ export class PipelineRegistry {
     return cloneDeep(this.artifactKinds.filter(k => k.isDefault));
   }
 
-  public getCustomArtifactKind(): IArtifactKindConfig {
```

Comment:
Not 100% clear why this needs to be removed when the new experience is intended to be flagged.

Reply:
it was completely unused except in tests

Reply:
effectively always returned `undefined` in production code

Reply:
Restored in 5ab4885a4.